subscriber/api.go

```diff
@@ -0,0 +1,54 @@
+package subscriber
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"go.uber.org/zap"
+
+	"github.com/gorilla/mux"
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/event"
+	"github.com/openfresh/plasma/pubsub"
+)
+
+type Api struct {
```

Comment:
__[Golint]__

type Api should be API


<!-- Comment by [SideCI](https://sideci.com) -->